spec/views/collections/_show_descriptions.html.erb_spec.rb

```diff
@@ -13,6 +13,7 @@
     end
     before do
       assign(:presenter, presenter)
+      allow(Sufia::CollectionSizeService).to receive(:run).and_return(123456678)
```

Comment:
Is this just here so that things don't blow up?  If so, maybe just end at `receive(:run)`?  Having the return value there suggests it's relevant for the tests but I'm guessing it's a garbage value.